models/card.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+
+const { Schema } = mongoose;
+const { ObjectId } = Schema.Types;
+
+const cardSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 30,
+  },
+
+  link: {
+    type: String,
+    validate: {
+      validator(link) {
+        return validator.isURL(link);
+      },
+      message: (url) => `${url.value} некорректный адрес!`,
+    },
+    required: true,
+  },
+
+  owner: {
+    type: { ObjectId },
+    ref: 'User',
+    required: true,
+  },
+
+  likes: {
+    type: [ObjectId],
+    ref: 'User',
+    default: [],
+  },
```

Comment:
Нужно исправить: некорректная схема поля.
Для того, чтобы объявить массив `ObjectId` нужно указывать так:
```suggestion
  likes: [{
    type: ObjectId,
    ref: 'User',
    default: [],
  }],
```